### PR TITLE
CarbonFootprint (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
brave_46EYhs7C3e
brave_TgJmS6P5Ah

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/CarbonFootprint/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,506 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CarbonFootprint",
+    "description": "Calculate the carbon footprint in various situations, such as travel and hydro usage. Carbon equivalent is the key unit used to report one's carbon footprint. This API enables developers to calculate carbon footprint without having to be concerned about the various conversion ratios.",
```
Comment (v-EgorKozhadei) [inline independent-publisher-connectors/CarbonFootprint/apiDefinition.swagger.json:5]: @troystaylor the word API is not allowed in description, please remove it.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @troystaylor! Thank you for your submission! Please take a look at one comment, we'll be good to go after the fix.

**Comment by v-EgorKozhadei** [comment]
[[certify-connector]]

**Comment by v-EgorKozhadei** [comment]
Hello @troystaylor, 

I hope you are doing well. 

Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.  

Please let us know if you have any questions. 

Thank you very much for working with us.

**Comment by v-EgorKozhadei** [comment]
Hello @troystaylor, 

I hope you are doing well. 

Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. 

Please let us know if you have any questions. 

Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**